src/H5Aint.c

```diff
@@ -104,6 +104,7 @@ const unsigned H5O_attr_ver_bounds[] = {
     H5O_ATTR_VERSION_3,     /* H5F_LIBVER_V18 */
     H5O_ATTR_VERSION_3,     /* H5F_LIBVER_V110 */
     H5O_ATTR_VERSION_3,     /* H5F_LIBVER_V112 */
+    H5O_ATTR_VERSION_3,     /* H5F_LIBVER_V114 */
```

Comment:
Similar to the H5Dlayout change, this looks a little suspect but I'm guessing it works out the same way as before

Reply:
Fixed